lib/phoenix/digester.ex

```diff
@@ -185,35 +185,43 @@ defmodule Phoenix.Digester do
   end
 
   defp digested_contents(file, manifest) do
-    if Path.extname(file.filename) == ".css" do
-      digest_asset_references(file, manifest)
-    else
-      file.content
+    case Path.extname(file.filename) do
```

Comment:
I think we have two options. Or we pattern match here and then call `css_asset_references` and `js_asset_references` or we call `digest_asset_references(Path.extname(file.filename), ...)` and pattern match inside each function clause. Today we are doing a mixture which is a bit unclear.